.github/actions/install-pyroscope/action.yaml

```
@@ -0,0 +1,32 @@
+name: InstallGrafana
```

```suggestion
name: InstallPyroscope
```

Match the GH action file name to the name in the file